src/ss_game_node.py

```diff
@@ -57,11 +58,15 @@ class ss_game_node():
 
     def __init__(self):
         """ Initialize anything that needs initialization """
+
         # set up logger
         self.logger = logging.getLogger(__name__)
         # configure logging
         try:
-            config_file = "ss_log_config.json"
+            __location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(sys.argv[0])))
```

Comment:
I'm not against getting the absolute path to the config files, but I am curious what problems you encountered with using relative paths for the config files and for the log file locations.